ui/src/app/cohort-search/list-search-v2/list-search-v2.component.tsx

```diff
@@ -412,7 +412,10 @@ export const ListSearchV2 = fp.flow(withCdrVersions(), withCurrentWorkspace())(
                 <th style={{...styles.columnHeader, textAlign: 'center', width: '12%'}}>View Hierarchy</th>
```

Comment:
Now that the header is separate from the body, we need to add the column widths used here to the `<td>`s in the `renderRow` function. Currently the columns in the header don't line up with the body.